modules/payloads/stagers/php/bind_tcp_ipv6.rb

```diff
@@ -4,15 +4,15 @@
 ##
 
 require 'msf/core'
-require 'msf/core/payload/php'
 require 'msf/core/handler/bind_tcp'
+require 'msf/core/payload/php/bind_tcp'
 
-module Metasploit3
+module Metasploit4
 
-  CachedSize = 1350
+  CachedSize = 1182
```

Comment:
Stagers reduced in size due to removal of code that does checking at runtime. Now that the payloads are generated on the fly, we can do that at generation time instead.